tests/area/test_coefficient_area.py

```diff
@@ -118,9 +119,10 @@ def test_calculate_community_after_meter_data(cls):
         assert isclose(scm._home_data[house2.uuid].energy_bought_from_community_kWh, 0.00)
         assert isclose(scm._home_data[house2.uuid].energy_sold_to_grid_kWh, 0.04)
 
-    @classmethod
-    def test_trigger_energy_trades(cls):
-        grid_area = cls._create_2_house_grid()
+    @staticmethod
+    @pytest.mark.skip("calculate_home_energy_bills should be revisit")
+    def test_trigger_energy_trades(_create_2_house_grid):
```

Comment:
This test was failing before this PR. For now it is marked as to skip and the new bug ticket was created to deal with it.

Reply:
Already fixed here https://github.com/gridsingularity/gsy-e/pull/1485